packages/frontend-sfe/src/lib/truncateHash.ts

```diff
@@ -1,3 +1,6 @@
-export const truncateHash = (hash: string, length = 38): string => {
-  return hash.replace(hash.substring(6, length), '...')
+export const truncateHash = (hash?: string, length = 38): string => {
+  if (!hash) return ''
+  let truncatedHash = hash
+  truncatedHash = truncatedHash.replace(truncatedHash.substring(6, length), '…')
+  return truncatedHash
```

Comment:
Is that necessary for making TS happy? Why can't you keep 
```suggestion
  return hash.replace(truncatedHash.substring(6, length), '…')
```
after the early exit? Wouldn't TS infer that `hash` is not undefined after the first line check?

Reply:
Changed in https://github.com/yieldgate/yieldgate/pull/67/commits/397110aa208e37a69503ffb1997997c94eebbced.

In another iteration, iIdid two things with `truncateHash` on multiple lines. There it made sense to have a dedicated variable.